### ISSUE TITLE
Separate initial "Ignore" message checks into separate function

### ISSUE THREAD

**Issue by chris-mcdonald-dev** [chris-mcdonald-dev/korean-discord-app#24]
<ins>**Task for New Contributors**</ins>

In `src/index.js` these if statements to ignore messages could be grouped into a **separate function**:

```js
client.on("message", (message) => {
	if (!message.guild) return; // Ignores DMs

	if (message.author.bot) {
		if (message.type === "PINS_ADD") message.delete();
		return; // Ignores messages from bots
	}
	if (message.type === "PINS_ADD") return; // Ignores PIN messages
```

Please make sure to remember to return from both the new function and the `client.on("message")` function if any of these conditions are met as well.

**Comment by abheepsapattnaik** [comment]
@chris-mcdonald-dev Can I pick this up?

**Comment by abheepsapattnaik** [comment]
@chris-mcdonald-dev Hey. Can I pick this up and give it a try?

**Comment by chris-mcdonald-dev** [comment]
Thanks again for doing this!
Fixed with PR #26. Closing.
